src/components/BuildingsListPage/buildingslist.js

```diff
@@ -0,0 +1,653 @@
+    }
+  },
+  displayContent: (currentPage) => {
+    document.querySelector('.building-list').innerHTML = buildingList.contentTemplate(currentPage)
+    const buildingListItems = document.querySelectorAll('.building-list__item')
+    buildingListItems.forEach(item => {
+      item.addEventListener('mouseenter', () => {
+        buildingList.hoverHandlerBuilding(event, currentPage)
+      })
+      item.addEventListener('click', () => {
+        const address = item.dataset.address
+        let currentAddress = ''
+        for (let i = 0; i < features.length; i++) {
+          // TODO REPLACE LATER BY REQUEST
+          const addressData = `${features[i].properties.street} ${features[i].properties.number} ${features[i].properties.zip_code} ${features[i].properties.city}`
+          if (address === addressData) {
+            currentAddress = features[i]
+          }
+        }
+        buildingList.showDetail([currentAddress])
+      })
+    })
+
+    buildingListItems.forEach(item => {
+      item.addEventListener('mouseleave', event => {
+        if (popup) {
+          popup.remove()
+        }
+      })
+    })
+  },
+
+  contentTemplate: (currentPage) => {
+    let html = ''
+    if (!moveMap) {
+      for (let i = (currentPage - 1) * itemsPerPage; i < (currentPage * itemsPerPage) && i < data.features.length; i++) {
+        html += `
+        
+        <li class="building-list__item" data-address="${data.features[i].properties.street} ${data.features[i].properties.number} ${data.features[i].properties.zip_code} ${data.features[i].properties.city}">
+            <div class="building-list__item__container">
+              <div class="building__img" style="background-image: url('${data.features[i].properties.image}');">
+              </div>
+                <div class="building__info">
+                    <div class="building__tags">`
+
+        if (data.features[i].properties.styles != null) {
+          html += `<div class="tag tag--style tag--small">${data.features[i].properties.styles}</div>`
+        }
+        if (data.features[i].properties.typographies != null) {
+          html += `<div class="tag tag--type tag--small">${data.features[i].properties.typographies}</div>`
+        }
+
+        if (data.features[i].properties.intervenants != null) {
+          html += `<div class="tag tag--architect tag--small">${data.features[i].properties.intervenants}</div>`
+        }
+
+        html += '</div>'
+
+        if (data.features[i].properties.name != null) {
+          html += ` <p class="building__name" >${data.features[i].properties.name}</p>`
+        }
+        html += `
+                    <p class="building__street">${data.features[i].properties.street} ${data.features[i].properties.number} </p>
+                    <p class="building__municipality"> ${data.features[i].properties.zip_code} ${data.features[i].properties.city}</p>
+                </div>
+            </div>
+        </li>
+        `
```

Comment:
This template seems to be used multiple times ; needs to be moved to a "sub-module"